src/Form/Examples.md

```diff
@@ -1,3 +1,38 @@
 ### Examples
 
-This component is a work in progress, we suggest you to ignore it for now.
+Simple wrapper for form that adds "submit on Enter" behavior
+
+```js
+initialState = {
+  firstName: "",
+  lastName: ""
+};
+
+const onChange = key => value => setState({ [key]: value });
+
+<Form.Form
+  column
+  submitLabel="submit"
+  renderSubmit={props => (
+    <StatefulButton
+      {...props}
+      style={{ ...props.style, width: "300px", marginLeft: "auto" }}
+    />
```

Comment:
what if we have a form with a submit and a cancel button, for example a modal. This feels a bit limited for those cases, doesn't it?

Reply:
Agreed, but you can always render
```ts
renderSubmit={submitProps => (
  <View><Cancel /><Submit {...submitProps} /></View>
)}
```

Reply:
fair enough.